### ISSUE TITLE
[Feature Request] Sync history between devices

### ISSUE THREAD

**Issue by SonOfDiablo** [fgl27/SmartTwitchTV#167]
It would be nice if you could sync your history between multiple devices so you can stop watching on one device and pickup on another. 

I know hosting a server for this could be expensive, so alternatively it could be done by saving the history to cloud storage solutions like dropbox/onedrive/etc. 

It's the only feature from the main twitch app I'm still missing after switching over.

**Comment by fgl27** [comment]
yes this can be added, using a Google account + driver backup or similar.

I can add it as a future feature but will not work on it right way as isn't a priority.
but is an open source app anyone can add features.

**Comment by fgl27** [comment]
done
